bundle/edu.gemini.p1backend/TODO.md

```diff
@@ -0,0 +1,50 @@
+# Short term
+
+Something that improves the situation now. Nothing for the future. A dead-end by
+design. Something to keep chugging away until we do it right. It shouldn't
+take more work than 2 weeks.
+
+1. Code lives in ocs.
+1. URLs:
+   - Receiver of P1 submissions: phase1.gemini.edu/receive/.../...
+   - Viewer of P1 submissions: phase1.gemini.edu/fetch/.../...
+1. Each backend a different path like now.
+1. Configuration file for each backend (that file is read with every new
+   submission). Used for prepending semester to Ids, close deadline, etc.
+1. Id counter from previous file names (hashes discussed but discarded).
```

Comment:
Yes, I'm referring to the counter. Is a counter the only way to assign ids? They could be random, like is done almost everywhere that I know of where ids are generated.

Why do they need to be in order really? I mean, it's nice that you can get an idea of which proposal came first but of all the people I asked about the backend servers you are the only one who has told me that's an important feature.

There is a ticket where they mention gaps in the ids but that's only confusing because an order is expected but if you don't expect an order the problem is solved.

For me no order is less work but if you tell me you've been told by the users that they need that ordering of ids, I'll take your word for it, and work something out.